docs/mkdocs/mkdocs.yml

```diff
@@ -250,6 +250,7 @@ nav:
         - 'JSON_HAS_RANGES': api/macros/json_has_ranges.md
         - 'JSON_HAS_THREE_WAY_COMPARISON': api/macros/json_has_three_way_comparison.md
         - 'JSON_NOEXCEPTION': api/macros/json_noexception.md
+        - 'JSON_DISABLE_ENUM_SERIALIZATION': api/macros/json_disable_enum_serialization.md
```

Comment:
This list is sorted, so with the name change, this should move up.

Reply:
Moved up, so it is sorted.